xmsannotator/tests/testthat/test-multilevelannotationstep2.R

```diff
@@ -0,0 +1,80 @@
+patrick::with_parameters_test_that(
+  "multilevelannotationstep2:",
+  {
+    # Arrange
+    testthat_wd <- getwd()
+    test_path <- file.path(
+      testthat_wd,
+      "test-data",
```

Comment:
```suggestion
      "testdata",
```

I think CI downloads the files to `testdata` directory, not `test-data`. Seems like this concerns `test_get_chemscore`, `test_integration_utils`, and `test-multilevelannotationstep` 2,3,4 (but needs to be double-checked globally).